src/System.Management.Automation/engine/remoting/fanin/PSSessionConfigurationData.cs

```diff
@@ -55,6 +51,11 @@ internal set
             }
         }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether Server Manager is enabled.
+        /// </summary>
```

Comment:
I am concerned about backward compatibility when converting a public field into a public property.

Reply:
Yes, I think we break a binary compatibility.